### DISCUSSION TITLE
How to configure new Workbox path

### DISCUSSION THREAD

**Discussion by damin-zhang** [reactplay/react-play#979] [Q&A]
I follow the blog 'React Play: Building a Progressive Web Application (PWA) using React' to configure PWA, but there is no value in my process.env.PUBLIC_URL variable, and there are other ways to import this file in the United States and Europe.

thanks for answering.

**Comment by atapas** [comment]
@frankiefab100 Check this out

**Reply by frankiefab100** [reply]
Alright

**Comment by atapas** [comment]
> I follow the blog 'React Play: Building a Progressive Web Application (PWA) using React' to configure PWA, but there is no value in my process.env.PUBLIC_URL variable, and there are other ways to import this file in the United States and Europe.
> 
> thanks for answering.

@damin-zhang you can join our Discord for more real-time answers and updates.

**Reply by damin-zhang** [reply]
Ok, is the name of the community react-play?

**Reply by atapas** [reply]
Here it is: https://discord.gg/UfYj4MvW9A

**Comment by frankiefab100** [comment]
> I follow the blog 'React Play: Building a Progressive Web Application (PWA) using React' to configure PWA, but there is no value in my process.env.PUBLIC_URL variable, and there are other ways to import this file in the United States and Europe.
> 
> thanks for answering.

Could you please share more details about this. May be a screenshot will help.
